scripts/check_changelog_format.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+# Check if a line matches any of the specified patterns
+def is_valid_line(line):
+    patterns = [
+        r"^Fixes:\s*.*$",
```

Comment:
I'm not sure if `python` is really warranted for this task
```
mkdir -p .unreleased;echo -e 'Fixes:\nImplements:\n junk\n Thax:' > .unreleased/file1;
if grep -vrEf <(echo -e '^Fixes:\n^Implements\n^Thanks:\n^$' ) .unreleased ; then echo mismatrch see above;else echo ok;fi
```

Reply:
Thought having in python will be easier to extend if needed.